Sources/Analytics/Onboarding/BiometricEnrollmentAnalytics.swift

```diff
@@ -1,13 +1,13 @@
 import GDSAnalytics
 import Logging
 
-enum BiometricEnrollmentAnalyticsScreen: String, ScreenType {
-    case faceIDEnrollment = "faceIDEnrollmentScreen"
-    case touchIDEnrollment = "touchIDEnrollmentScreen"
+enum BiometricEnrolmentAnalyticsScreen: String, ScreenType {
+    case faceIDEnrolment = "faceIDEnrollmentScreen"
+    case touchIDEnrolment = "touchIDEnrollmentScreen"
     case unlockScreen = "unlockScreen"
 }
 
-enum BiometricEnrollmentAnalyticsScreenID: String {
-    case faceIDEnrollment = "7590abba-f48b-4790-be64-f6ffc3dd35e2"
-    case touchIDEnrollment = "cc1db40a-2a5a-43fe-a1fe-fc85b55fdcab"
+enum BiometricEnrolmentAnalyticsScreenID: String {
+    case faceIDEnrolment = "7590abba-f48b-4790-be64-f6ffc3dd35e2"
+    case touchIDEnrolment = "cc1db40a-2a5a-43fe-a1fe-fc85b55fdcab"
 }
```

Comment:
Just out of curiosity, how come `unlockScreen` doesn't have a screenID? 
Also, since we're renaming enrolment, don't forget the file name! 😅

Reply:
It's a good point and based on [this ticket](https://govukverify.atlassian.net/browse/DCMAW-8394), it feasibly should do although looks like some shuffling of the tickets happened. I will add it for consistency's sake.
Good spot 🙂